js/about/passwords.js

```diff
@@ -63,7 +63,7 @@ class SiteItem extends React.Component {
   }
 
   onDelete () {
-    aboutActions.deletePasswordSite(this.props.site)
+    aboutActions.deletePassword(this.props.site.toJS())
```

Comment:
it seems that `onDelete` here should then remove the `blacklisted_by_user` attribute, not delete the password